LICENSE.txt

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2011, Donald Stufft
```

Comment:
@alex-ber, WHAT is this file ?

Reply:
This is license file auto-generated by github. Do you want me to remove it?

Reply:
Done, anyway.